libs/ECS/Components/VelocityComponent.hpp

```diff
@@ -0,0 +1,32 @@
+/*
+** EPITECH PROJECT, 2022
+** Project
+** File description:
+** VelocityComponent
+*/
+
+#ifndef VELOCITYCOMPONENT_HPP_
+#define VELOCITYCOMPONENT_HPP_
+
+#include "Component.hpp"
+
+namespace ecs
+{
+    /// @brief The velocity class, multiplying the movement of the entity
+    class Velocity : public Component{
+      public:
+        /// @brief The multiplier of the movement (between 0.1 and 10)
+        double multiplier;
+
+        /// @brief Constructor of the Velocity Component
+        /// @param newMultiplier the movement's multiplier
+        Velocity(double newMultiplier) : multiplier(newMultiplier) {};
+        /// @brief Copy Constructor of the Velocity Component
+        /// @param old the copy of the Velocity Component
+        Velocity(const Velocity &old) : multiplier(old.multiplier) {};
+        /// @brief Default Destructor
+        ~Velocity() = default;
```

Comment:
For this component, I would see a vector because an entity can be faster on the X axis than the Y axis. And vice versa.